setup.py

```diff
@@ -64,7 +72,7 @@ def __init__(self, *args, **kwargs):
     from distutils.command.build_ext import build_ext
     if not all(os.path.exists(path) for path in [
         compress_source, crypto_source, chunker_source, hashindex_source,
-        platform_linux_source, platform_freebsd_source]):
+        platform_linux_source, platform_freebsd_source]) and not on_rtd:
```

Comment:
move the "not on_rtd" to the beginning, it easier to read and more efficient also.